docs/features/default.test.md

```diff
@@ -0,0 +1,173 @@
+﻿# General concerns
+- [ ] Parsing: `default < e1 > e2`
+- [ ] Backward compatibility: report version error with earlier `/langversion`
```

Comment:
Covered by `TestCSharp7`
`error CS8107: Feature 'default literal' is not available in C# 7. Please use language version 7.1 or greater.`